src/PJ_merc.c

```diff
@@ -48,11 +49,36 @@ static LP s_inverse (XY xy, PJ *P) {           /* Spheroidal, inverse */
     return lp;
 }
 
+/* Cf http://desktop.arcgis.com/en/arcmap/10.3/guide-books/map-projections/mercator.htm */
+#define AUX_SPHERE_TYPE_SEMI_MAJOR 0
+#define AUX_SPHERE_TYPE_SEMI_MINOR 1
+#define AUX_SPHERE_TYPE_AUTHALIC_RADIUS 2
+/* Unimplemented #define AUX_SPHERE_TYPE_AUTHALIC_LATITUDE 3 */
```

Comment:
healpix gives some hints as to how to implement this: https://github.com/OSGeo/proj.4/blob/master/src/PJ_healpix.c#L225-L246